### ISSUE TITLE
large default browser font size breaks office layout

### ISSUE THREAD

**Issue by zebapy** [middlebury/midd-frontend#129]
![Screen Shot 2019-05-13 at 9 47 38 AM](https://user-images.githubusercontent.com/2199845/57631360-5cc7e500-756d-11e9-8fe8-adcc84a686db.png)

You’re using Firefox 69.
Operating systemOS X 10.14
IP address 140.233.90.99
Javascript enabledY es
Cookies enabled Yes
Flash version 32.0.0
Java version Not installed
Websockets supported Yes
WebGL supported Yes
Language en-US
Browser size 1394 x 876
Screen size 1920 x 1200
Color depth 24 bit
AdBlock enabled Yes

**Comment by zebapy** [comment]
I cannot seem to reproduce this at the moment. 

<img width="644" alt="Screenshot 2019-05-15 11 01 01" src="https://user-images.githubusercontent.com/2199845/57786006-bd3b5b80-7700-11e9-8111-8abe4acb6aa6.png">

**Comment by zebapy** [comment]
I contacted Leanne since I could not recreate the issue and she mentioned her default browser font size is 24px. After I applied this to my testing browser, I had the same problem. 

![Screenshot 2019-11-15 11 36 52](https://user-images.githubusercontent.com/2199845/68959542-3bee7f80-079c-11ea-87a4-ea2e1466e0b7.png)


I'll need to dig into our styles to see how to handle spacing when default font size is changed since this is absolutely something we should accommodate.
